my-tweet/src/components/right-sidebar.js

```diff
@@ -0,0 +1,41 @@
+import { Box, Button, Flex, Heading, useDisclosure, Wrap } from "@chakra-ui/react"
+import axios from "axios"
+import { useEffect } from "react"
+import { usePost } from "../context/post-context"
+import { followHandler } from "../util-functions/follow-handler"
+import { unfollowHandler } from "../util-functions/unfollow-handler"
+import { UnfollowModal } from "./unfollow-modal"
+
+export const RightSidebar=()=>{
+    const {allUsers,setAllUsers,userFollow,setUserFollow,userUnfollow,setUserUnfollow}=usePost()
+    const { isOpen, onOpen, onClose } = useDisclosure()
+
+    const getAllUser=async()=>{
```

Comment:
suggestion: you can make separate function for api call so that your code looks more clean

Reply:
ok will do it